de.dlr.sc.virsat.uiengine.ui/src/de/dlr/sc/virsat/uiengine/ui/editor/snippets/AUiSnippetGenericTable.java

```diff
@@ -715,7 +715,7 @@ private CategoryAssignment getSelectedCategoryAssigment(Object selectedObject) {
 			ComposedPropertyInstance selectedCpi = (ComposedPropertyInstance) selectedObject;
 			ATypeInstance referencedTypeInstance = selectedCpi.getTypeInstance();
 

```

Comment:
// Please add a comment why the instanceof check is not needed anymore

Reply:
Instead simplified the code further since no check at all is needed here.